### DISCUSSION TITLE
Some diagnostic information is not displayed

### DISCUSSION THREAD

**Discussion by ofzo** [helix-editor/helix#10739] [General]
The same applies to rust-analyzer as an lsp, which can display more diagnostic information in vim

helix:
<img width="974" alt="image" src="https://github.com/helix-editor/helix/assets/18000037/d19c17ec-2723-41c8-8eaa-b2691b5e9eca">

vim:
![image](https://github.com/helix-editor/helix/assets/18000037/71b6bffb-19da-4362-b432-cd56c305d408)

**Comment by dead10ck** [comment]
Does helix display any diagnostics at all for you?

**Reply by ofzo** [reply]
yes, some but not all, can be displayed

![image](https://github.com/helix-editor/helix/assets/18000037/7b0c84db-aa59-4e7b-896e-58d812207266)

**Reply by ofzo** [reply]
It seems that only errors and warnings are displayed, and no recommendations are displayed
